{"name": "Successful login", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\t(No symbol) [0x00DA37D3]\n\t(No symbol) [0x00D38B81]\n\t(No symbol) [0x00C3B36D]\n\t(No symbol) [0x00C6D382]\n\t(No symbol) [0x00C6D4BB]\n\t(No symbol) [0x00CA3302]\n\t(No symbol) [0x00C8B464]\n\t(No symbol) [0x00CA1215]\n\t(No symbol) [0x00C8B216]\n\t(No symbol) [0x00C60D97]\n\t(No symbol) [0x00C6253D]\n\tGetHandleVerifier [0x0101ABF2+2510930]\n\tGetHandleVerifier [0x01048EC1+2700065]\n\tGetHandleVerifier [0x0104C86C+2714828]\n\tGetHandleVerifier [0x00E53480+645344]\n\t(No symbol) [0x00D40FD2]\n\t(No symbol) [0x00D46C68]\n\t(No symbol) [0x00D46D4B]\n\t(No symbol) [0x00D50D6B]\n\tBaseThreadInitThunk [0x767E00F9+25]\n\tRtlGetAppContainerNamedObjectPath [0x779F7BBE+286]\n\tRtlGetAppContainerNamedObjectPath [0x779F7B8E+238]\n\n", "trace": "  File \"c:\\repo\\selenium\\maquina_uno\\mv1\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\repo\\selenium\\maquina_uno\\mv1\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\steps_login.py\", line 58, in step_impl\n    obtener_mensaje_error(dashboard,\"Existe\")\n  File \"features\\steps\\steps_login.py\", line 20, in obtener_mensaje_error\n    e1 = f.selectoxpath(xpath)\n  File \"C:\\Repo\\Selenium\\Maquina_uno\\MV1\\Login\\features\\Base_page.py\", line 31, in selectoxpath\n    val = WebDriverWait(self.driver, 4).until(EC.presence_of_element_located((By.XPATH, elemento)))\n  File \"c:\\repo\\selenium\\maquina_uno\\mv1\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I am on the login page", "status": "passed", "start": 1678811047476, "stop": 1678811054063}, {"name": "When I enter my valid username and password", "status": "passed", "start": 1678811054063, "stop": 1678811055455}, {"name": "And I click on the login button", "status": "passed", "start": 1678811055456, "stop": 1678811056013}, {"name": "Then I should see my personal dashboard", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\t(No symbol) [0x00DA37D3]\n\t(No symbol) [0x00D38B81]\n\t(No symbol) [0x00C3B36D]\n\t(No symbol) [0x00C6D382]\n\t(No symbol) [0x00C6D4BB]\n\t(No symbol) [0x00CA3302]\n\t(No symbol) [0x00C8B464]\n\t(No symbol) [0x00CA1215]\n\t(No symbol) [0x00C8B216]\n\t(No symbol) [0x00C60D97]\n\t(No symbol) [0x00C6253D]\n\tGetHandleVerifier [0x0101ABF2+2510930]\n\tGetHandleVerifier [0x01048EC1+2700065]\n\tGetHandleVerifier [0x0104C86C+2714828]\n\tGetHandleVerifier [0x00E53480+645344]\n\t(No symbol) [0x00D40FD2]\n\t(No symbol) [0x00D46C68]\n\t(No symbol) [0x00D46D4B]\n\t(No symbol) [0x00D50D6B]\n\tBaseThreadInitThunk [0x767E00F9+25]\n\tRtlGetAppContainerNamedObjectPath [0x779F7BBE+286]\n\tRtlGetAppContainerNamedObjectPath [0x779F7B8E+238]\n\n", "trace": "  File \"c:\\repo\\selenium\\maquina_uno\\mv1\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\repo\\selenium\\maquina_uno\\mv1\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\steps_login.py\", line 58, in step_impl\n    obtener_mensaje_error(dashboard,\"Existe\")\n  File \"features\\steps\\steps_login.py\", line 20, in obtener_mensaje_error\n    e1 = f.selectoxpath(xpath)\n  File \"C:\\Repo\\Selenium\\Maquina_uno\\MV1\\Login\\features\\Base_page.py\", line 31, in selectoxpath\n    val = WebDriverWait(self.driver, 4).until(EC.presence_of_element_located((By.XPATH, elemento)))\n  File \"c:\\repo\\selenium\\maquina_uno\\mv1\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1678811056013, "stop": 1678811071788}], "start": 1678811047475, "stop": 1678811071789, "uuid": "3c8bcee2-f22f-46b2-9f86-b52e1d9286d4", "historyId": "82e6fd88a69b5968ce989053b7ddc5a8", "fullName": "Login: Successful login", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}